src/main/java/com/epam/page/object/generator/validators/UniquenessLocatorValidator.java

```diff
@@ -40,7 +43,21 @@ public boolean isValid(SearchRule searchRule, ValidationContext validationContex
     }
 
     @Override
-    public String getExceptionMessage() {
-        return "This element is not uniqueness";
+    public String getExceptionMessage(SearchRule searchRule, ValidationContext validationContext) {
+       // return "This element is not uniqueness";
+        return validationContext.getUrls()
+                .stream()
+                .map(url -> {
+                    try {
```

Comment:
7-lines lambda is a bad style. Let's extract a method

Reply:
just extract lambda to method and use it inside method getExceptionMessage()???
or exctract some part of lambda?